### ISSUE TITLE
[BUG] an extra empty line added with echo=true

### ISSUE THREAD

**Issue by shg** [JunoLab/Weave.jl#400]
With doctype="multimarkdown", by weaving

~~~
```julia; echo=true
1
```
~~~

I get

~~~
```julia

1
```


```
1
```


~~~

You see an extra empty line before "1" in the echoed output. If I add a semicolon like

~~~
```julia; echo=true
1;
```
~~~

then the result does not contain the extra empty line.

~~~
```julia
1;
```


~~~

The extra empty line seems to be rendered with the other doctypes too (I haven't checked all of them though).

**Comment by andrew-jessop** [comment]
I have the same problem.
- Julia 1.5.2 
- Weave v0.10.5
- Linux Mint 20 and Debian testing (occurs on both)

**Comment by aviatesk** [comment]
will be fixed in v0.10.6
